mayastor/src/bdev/nexus/nexus_bdev_children.rs

```diff
@@ -217,7 +217,8 @@ impl Nexus {
             });
         }
 
-        self.cancel_child_rebuild_jobs(uri).await;
+        let cancelled_rebuilding_children =
+            self.cancel_child_rebuild_jobs(uri).await;
 
         let idx = match self.children.iter().position(|c| c.name == uri) {
             None => return Ok(()),
```

Comment:
if we return here then we don't restart the rebuilds? though I guess that's ok as we wouldn't have cancelled anything if the child is not present

Reply:
Agreed - if the child isn't present we shouldn't be using it in any rebuilds.